src/skins/vector/css/matrix-react-sdk/structures/_FilePanel.scss

```diff
@@ -15,13 +15,8 @@ limitations under the License.
 */
 
 .mx_FilePanel {
-    -webkit-box-ordinal-group: 2;
```

Comment:
Ideally yes, the postcss transition would be separate PR to the factoring out colours, and stripping out vendor prefixes, and theming.  Unfortunately they all grew organically at the same time and I'm not convinced it's worth the faff to untangle them now.  Future changes should be more reviewable.